### ISSUE TITLE
Change location of ~/bin/termux-url-opener

### ISSUE THREAD

**Issue by kutsan** [termux/termux-app#329]
It's just a simple request. I'd like to change the location of `~/bin/termux-url-opener` script. And even maybe its name too.

**Comment by ujeropoc** [comment]
I would have expected it in `.termux/url-opener` for consistency with Termux:boot or termux:tasker

**Comment by 46478** [comment]
> It's just a simple request. I'd like to change the location of `~/bin/termux-url-opener` script. And even maybe its name too.

hel me

**Comment by agnostic-apollo** [comment]
Isn't a symlink enough for this? Termux automatically follows symlinks when checking [if file exists](https://github.com/termux/termux-app/blob/v0.113/app/src/main/java/com/termux/filepicker/TermuxFileReceiverActivity.java#L183), due to `isFile()`. Unless you don't want to add the `bin` directory, in which case I guess could be added in Termux Settings.

```
mkdir -p ~/bin
ln -sf ~/url.sh ~/bin/termux-url-opener
echo '#!/data/data/com.termux/files/usr/bin/bash
echo "$*"
sleep 5' > ~/url.sh
chmod +x ~/url.sh
```

**Comment by kutsan** [comment]
I think all config files about Termux should live inside `~/.termux` directory (although I prefer `~/.config/termux` as for the XDG directory spec). So, symlinking works and that's what I've been using this whole time but making a certain directory default would be better.

**Comment by agnostic-apollo** [comment]
>I think all config files about Termux should live inside ~/.termux directory

I would agree with this but can't change the default now. It would break existing setups and then we will get "spammed" with issues for weeks, months.

Adding path in `termux.properties` would be even better, since it would automatically be backed up with `tar` backups.

**Comment by kutsan** [comment]
You don't need to drop support for old directories, you can still change the default and encourage people to use the new default but still support the old directories like you used to do.

**Comment by agnostic-apollo** [comment]
A multi `if` conditional would be required anyways for custom path, so old and new default can be added too.
